packages/css-data/src/property-parsers/transition-property-extractor.ts

```diff
@@ -12,19 +11,19 @@ export const isTimingFunction = (timing: string) => {
 };
 
 export type ExtractedTransitionProperties = {
-  property?: KeywordValue | null;
-  timing?: KeywordValue | null;
-  delay?: StyleValue | null;
-  duration?: StyleValue | null;
+  property?: KeywordValue;
+  timing?: KeywordValue;
+  delay?: UnitValue;
+  duration?: UnitValue;
 };
 
 export const extractTransitionProperties = (
   transition: TupleValue
 ): ExtractedTransitionProperties => {
-  let property: KeywordValue | null = null;
-  let timing: KeywordValue | null = null;
-  let duration: UnitValue | null = null;
-  let delay: UnitValue | null = null;
+  let property: KeywordValue | undefined = undefined;
```

Comment:
`= undefined` is not required when type explicitly specifies it

Reply:
`ts` was complaining for me with `Used before delaration type error`. So just initialised with `undefined`